backend/internal/storage/mongo/storage.go

```diff
@@ -0,0 +1,42 @@
+package mongo
+
+import (
+	"backend/config"
+	"backend/internal/storage"
+	"context"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func ConnectDatabase(config config.DB) *mongo.Client {
```

Comment:
```suggestion
func ConnectDatabase(ctx context.Context, config config.DB) (*mongo.Client, error) {
```
provides more flexibility to the caller